pkgs/development/python-modules/sphinxcontrib-moderncmakedomain/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy27
+, hatchling
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-moderncmakedomain";
+  version = "3.27.0";
+  disabled = isPy27;
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "sphinxcontrib_moderncmakedomain";
+    hash = "sha256-UeJZ6R9Y0XzA+skwf9QBBqpZ1ayqdBiHkD/DZgNh0aE=";
+  };
+
+  nativeBuildInputs = [
+    hatchling
+    sphinx
+  ];
+
+  pythonNamespaces = [ "sphinxcontrib" ];
+
+  meta = with lib; {
+    description = "sphinxcontrib-moderncmakedomain is a Sphinx extension which renders CMake documentation.";
+    homepage = "https://github.com/scikit-build/moderncmakedomain";
+    license = licenses.bsd3;
+    maintainers = teams.sphinx.members;
```

Comment:
This probably belongs more to the scikit people than the general sphinx

Reply:
then I am not sure but sphinx members should not receive this.

Reply:
I set myself as the maintainer! I'll keep it updated.